### DISCUSSION TITLE
Trained a Mask Coverage Detection model that can detect improper wear

### DISCUSSION THREAD

**Discussion by sagar-singh20** [cabani/MaskedFace-Net#8] [Show and tell]
The dataset was really helpful in achieving my goal of creating an object detection based model that can identify if a person is not wearing his/her masks properly. More improvements are underway but then I am sure this vast dataset will help many in building something really helpful especially when a new strain of the virus is in place. 

Thanks to the entire community of MaskedFace-Net

**Comment by haifa33** [comment]
hi bro,
can i ask u if you created an annotation file for the MaskedFace-Net dataset? and u can share it with me
or if there was already exist file made by the author himself ? because i did not find any
i will be grateful, thank u

**Comment by cabani** [comment]
Dear @haifa33,

Each image of the MaskedFace-Net dataset has its corresponding file naming in the FFHQ dataset. Hence, the metadata file “ffhq-dataset-v2.json” (see the FFHQ webpage) can be exploited for your processing of MaskedFace-Net.

Regards,
Maskedface-Net Team

**Comment by haifa33** [comment]
Thanks a million for this wonderful team.

Regards the metadata file, should i change the image labels? In order to match (mask & no mask) classes?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
